add the option not to set an alias

jograb opened the pull request:
ℹ️ https://github.com/RocketChat/Rocket.Chat/issues/22218 (if you want to avoid setting the global bot permissions, one option is to disable the alias. this should fix the "Not enough permission"-error)

AmShaegar13 commented on lib/xify/output/rocket_chat.rb:
```diff
@@ -10,17 +10,25 @@ class RocketChat < Base::RocketChat
+          else
+            $alias = nil
+            $title = "#{event.args[:parent]} by #{event.author}"
+          end
```
Please don't use $ to start variable names in ruby. Those names are reserved for global variables.

```suggestion
          if @config['alias']
            author = event.author
            title = event.args[:parent]
          else
            author = nil
            title = "#{event.args[:parent]} by #{event.author}"
          end
```

AmShaegar13 commented on lib/xify/output/rocket_chat.rb:
```diff
@@ -10,17 +10,25 @@ class RocketChat < Base::RocketChat
+
+            req.body = {
+              channel: @config['channel'],
+              alias: "#{$alias}",
```
Unnecessary use of string interpolation.
```suggestion
              alias: author,
```

AmShaegar13 commented on lib/xify/output/rocket_chat.rb:
```diff
@@ -10,17 +10,25 @@ class RocketChat < Base::RocketChat
+              alias: "#{$alias}",
+              attachments: [
+                {
+                  title: "#{$title}",
```
Unnecessary use of string interpolation.
```suggestion
                  title: title,
```

AmShaegar13 commented:
Hey, thanks for your contribution. Do you mind fixing my suggestions and squash your commits?

Commit:
add the option not to set an alias

ℹ️ https://github.com/RocketChat/Rocket.Chat/issues/22218 (if you want to avoid setting the global bot permissions, one option is to disable the alias. this should fix the "Not enough permission"-error)

AmShaegar13 commented:
I created version [0.3.0](https://rubygems.org/gems/xify-rocket_chat/versions/0.3.0).
